cmd/kubelet/app/options/options.go

```diff
@@ -272,4 +274,6 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.EvictionPressureTransitionPeriod.Duration, "eviction-pressure-transition-period", s.EvictionPressureTransitionPeriod.Duration, "Duration for which the kubelet has to wait before transitioning out of an eviction pressure condition.")
 	fs.Int32Var(&s.EvictionMaxPodGracePeriod, "eviction-max-pod-grace-period", s.EvictionMaxPodGracePeriod, "Maximum allowed grace period (in seconds) to use when terminating pods in response to a soft eviction threshold being met.  If negative, defer to pod specified value.")
 	fs.Int32Var(&s.PodsPerCore, "pods-per-core", s.PodsPerCore, "Number of Pods per core that can run on this Kubelet. The total number of Pods on this Kubelet cannot exceed max-pods, so max-pods will be used if this calculation results in a larger number of Pods allowed on the Kubelet. A value of 0 disables this limit.")
+	fs.StringVar(&s.KernelTunableBehavior, "kernel-tunable-behaviour", s.KernelTunableBehavior, "Default kubelet behaviour for kernel tunning. Valid values: warning, modify, error")
```

Comment:
Each and every option we add needs to ideally be tested. The lesser options we add, the easier it is to maintain in general.
If a need arises to support `warning`, we can consider it then..
As @derekwaynecarr pointed out, it will be difficult to track the various kernel configs that kubelet will depend on over time.